subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/accessors/PluginAccessorsClassPath.kt

```diff
@@ -259,6 +259,7 @@ fun BufferedWriter.appendSourceCodeForPluginAccessors(
                     /**
                      * The `$id` plugin group.
                      */
+                    @org.gradle.api.Generated
```

Comment:
:x: this isn't necessary anymore now that the annotation is on `@file:`

Once removed the same can be removed in assertions in `PluginAccessorsClassPathTest`

Reply:
It is necessary, JaCoCo marked those as not covered, don't ask my why.

Reply:
Ok, this is because the annotation `@file` ends up on the `...Kt.class` but those classes defined in the file end up into separate `Name.class` files and JaCoCo can't make sense of the `@file:` scope.